README.md

```diff
@@ -9,9 +9,13 @@ To add a conference to the list, please [contribute](#contributing)!
 | Conference | Date | Venue | Description |
 |------------|------|-------|-------------|
 | [PyCon India](https://in.pycon.org/2018/) | 05-09 Oct | Hyderabad International Convention Centre (HICC), Hyderabad | The premier conference in India on using and developing the Python programming language. |
-| [JSFoo](https://jsfoo.in/2018/) | 13-14 Sep | Bengaluru | The eighth edition of India’s first JavaScript conference.
+| [JSFoo](https://jsfoo.in/2018/) | 13-14 Sep | Bengaluru | The eighth edition of India’s first JavaScript conference.|
+| [ODSC (One Data Science Conference) ](https://2018.seleniumconf.in/) | 30 Aug – 2 Sep | Hotel Lalit Ashok, Bengaluru  | Conference on AI and Data Science |
```

Comment:
Like @thayumaanavan mentioned, can you please correct this link?

Happy to merge after this is done!